mmv1/products/kms/terraform.yaml

```diff
@@ -98,6 +98,24 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       decoder: templates/terraform/decoders/kms.go.erb
       encoder: templates/terraform/encoders/kms_crypto_key.go.erb
       update_encoder: templates/terraform/update_encoder/kms_crypto_key.go.erb
+  CryptoKeyVersion: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+
+      Destroying a cryptoKeyVersion will not delete the resource from the project.
+    id_format: "{{name}}"
+    import_format: ["{{name}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "kms_crypto_key_version_basic"
+        primary_resource_id: "example-key"
+        skip_test: true
+    properties:
+      state: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      custom_delete: templates/terraform/custom_delete/kms_crypto_key_version.erb
+      custom_import: templates/terraform/custom_import/kms_crypto_key_version.go.erb
```

Comment:
The import code should also set the `crypto_key` field, similar to how `resourceKMSCryptoKeyImport` sets `key_ring`. (google_kms_crypto_key_version.crypto_key is a parent field, similar to google_kms_crypto_key.key_ring.)

Reply:
After adding crypto_key field to the import code, at import state phase we start to get **"projects/tf-test-2868264652724940424/locations/us-central1/keyRings/tf-test-67qpots2pr/cryptoKeys/"** The ID is still missing, I will try to see how to add that.

Reply:
Solved for now.